Source/AwsCommonRuntimeKit/crt/CRTErrorGenerated.swift

```diff
@@ -0,0 +1,307 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0.
+// This file is generated using Script/CRTErrorGenerator.swift.
+// Do not modify this file.
+
+import AwsCCommon
+
+/// Error type for CRT errors thrown from C code
+public enum CRTError: Int32, Error {
+
+    case UNKNOWN_ERROR_CODE = -1
+
+    /// AWS-C-COMMON
+    case AWS_ERROR_SUCCESS = 0
```

Comment:
I would be happy to omit SUCCESS from this list

Reply:
What should we do if there is a bug where we throw the error but error code is zero? Should we just fatal exit or throw errorSuccess or unknownErrorCode?

Reply:
If the error-code is 0 and we need to throw the error, we should turn it into `AWS_ERROR_UNKNOWN`

Reply:
Fixed, thanks.